### PR TITLE
"Basics of XNCP" Guide

### PR DESCRIPTION
**Pull request by NextinMono**
I wrote a general introduction to XNCP editing, and some of the structures of the file. Definitely needs a touch up, but I hope I did well enough to satisfy the guidelines!

### CODE REVIEW

**Comment by PTKay** [comment]
As per the guidelines:
> Guides should include an info box before the title, listing the tools that they're using. The info box should include a link to the relevant tools page

You can check the [guide for XNCP animations](https://github.com/HedgeDocs/HedgeDocs.github.io/blob/main/docs/guides/hedgehog-engine/blueblur/files/xncp/anims.md) for an example on how to setup a guide

**Review by PTKay — CHANGES_REQUESTED** [review]
Include info box with the necessary tools

**Comment by NextinMono** [comment]
On it!

**Comment by NextinMono** [comment]
Should be changed now, see if it's okay now

File: docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+description: XNCP Basics, using 010 Editor
+---
+!!! info
+    This guide uses the following tools:
+
+    - 010 Editor (with XNCP Template)
+
+    These tools and more can be found in the [XNCP tools page](/tools/hedgehog-engine/blueblur/files/xncp)
+# The Basics of XNCP Editing
```
Comment (PTKay) [inline docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md:10]: Have you tested this locally btw? Not sure if you need an extra line between the info box and the page title...

Comment (NextinMono) [reply docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md:10]: I have tested it locally, with or without the space between info and title, it looks the exact same!

Comment (NextinMono) [reply docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md:10]: image

Comment (PTKay) [reply docs/guides/hedgehog-engine/blueblur/files/xncp/basics.md:10]: Nice so it's working properly. Seems good, will merge

**Review by PTKay — APPROVED** [review]

**Pull request merged**